src/ocean/text/convert/Integer.d

```diff
@@ -30,6 +30,138 @@ import ocean.transition;
 
 import ocean.core.Traits;
 
+import ocean.core.array.Search : find;
+
+import ocean.util.ReusableException;
+
+import ocean.math.Math;
+
+
+/*******************************************************************************
```

Comment:
I don't think this function should go first in the module. It's less commonly useful than `toInteger`, say.